hermes-tracker-elasticsearch/build.gradle

```diff
@@ -5,15 +5,12 @@ dependencies {
     implementation project(':hermes-common')
     implementation project(':hermes-tracker')
     implementation 'org.slf4j:slf4j-api:2.0.13'
-    // TODO: client has to have the same major version as backend: https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/client.html
-    // we need to use test-containers for elasticsearch server to able to use newer client.
-    api group: 'org.elasticsearch.client', name: 'transport', version: '6.8.12'
+    api group: 'org.elasticsearch.client', name: 'transport', version: '7.10.2'
```

Comment:
Newest [7.17.25](https://mvnrepository.com/artifact/org.elasticsearch.client/transport/7.17.25)

Reply:
Thats true, but it is suggested to use the same client version as cluster version.